plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardProjectConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+/**
+ * Collects all data needed to create and configure an App Engine Standard Project.
+ */
+class AppEngineStandardProjectConfig {
+
+  private String appEngineProjectId;
+  private String eclipseProjectName;
+  private String packageName;
+  private String eclipseProjectDirectory;
+
+  // todo does builder pattern make more sense here?
```

Comment:
Looking forward, I also wondered how we'd integrate AE-Flex, GKE, and GCE here.  And whether a deployment region is needed too (that seems to be specified on project creation).